share/cmake/scrimmage/GenerateSetEnv.cmake

```diff
@@ -2,11 +2,11 @@ include(CMakeParseArguments)
 
 function(GenerateSetEnv)
   set(options)
-  set(oneValueArgs SETUP_LOCAL_CONFIG_DIR LOCAL_CONFIG_DIR SETENV_IN_FILE)
+  set(oneValueArgs SETUP_HOME_CONFIG LOCAL_CONFIG_DIR SETENV_IN_FILE)
```

Comment:
Can you change the SETUP_HOME_CONFIG variable name back to SETUP_LOCAL_CONFIG_DIR? The SETUP_LOCAL_CONFIG_DIR option is directly related to the LOCAL_CONFIG_DIR, the LOCAL_CONFIG doens't necessarily have to be located in the user's $ENV{HOME}, but it defaults to HOME if the LOCAL_CONFIG_DIR isn't specified.